src/components/credit-card-number/code/templates/creditCardNumberTemplate.jsx

```diff
@@ -0,0 +1,52 @@
+    }
+
+    images = ( <div className="credit-card-images">{visa} {mastercard} {amex} {discover}</div> );
+  }
+
+  return (
+    <div className="component-credit-card-number">
+      {images}
+      <div className="credit-card-number">
+        {label}
+        <input
```

Comment:
Can we make use of the pattern attribute in HTML5 to help out here?: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#attr-pattern